web/settings/settings.go

```diff
@@ -192,6 +196,42 @@ func (h *HTTPHandler) getEmailConfirmation(c echo.Context) error {
 	}
 }
 
+func (h *HTTPHandler) installApp(c echo.Context) error {
```

Comment:
This name is a bit vague as we could be talking about any app. Something like `installFlagshipApp` would be clearer.